src/app/core/public/api/impl/Wire.ts

```diff
@@ -24,10 +25,37 @@ export class WireImpl<
             .unwrap();
     }
 
+    protected abstract get nodeKind(): string;
```

Comment:
assuming this is temporary based on TODO below, and will be moved somewhere else as a part of that?  (i.e. `ObjectInfo.nodeKind(): string` or eventually something more general that includes busses.